extensions/grpc/deployment/pom.xml

```diff
@@ -29,6 +29,11 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-grpc</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
```

Comment:
You should be able to depend on io.quarkus.security:quarkus-security, which should contain the interfaces etc you need.

Reply:
done